python/examples/integrations/flask_streaming/.dockerignore

```diff
@@ -3,4 +3,4 @@ __pycache__
 download_iris.sh
 train.py
 train.ipynb
-readme.md
+readme.md
```

Comment:
Interesting, it's saying the change is to have no newline?

Reply:
Interesting, so that actually removed the .dockerignore from the change. I guess it did have the end of file then?

Reply:
Right, I think the line ending is there and so it is ok as is, the important thing would be to exclude this file from pre-commit checks. I have no edits to this file in the latest comits.